src/linear_combination.jl

```diff
@@ -35,25 +35,23 @@ function apply(x::AbstractVector, LC::LinearCombination; inds=Colon())
 end
 
 """
-    apply(x::AbstractMatrix, LC::LinearCombination; dims=1, inds=Colon())
+    apply(A::AbstractMatrix, LC::LinearCombination; dims=2, inds=Colon())
```

Comment:
changed `x` to `A` more on a whim but also so I could use `x` in the `mapslices` below